deploy/openshift/deploy-kiali-to-openshift.sh

```diff
@@ -267,21 +266,26 @@ apply_dashboard() {
   apply_yaml ${yaml_path} ${yaml_url}
 }
 
-# Now deploy all the Kiali components to OpenShift
-# If we are missing one or more of the yaml files, download them
+# Now deploy all the Kiali components to OpenShift.
+# If we are missing one or more of the yaml files, download them.
 echo "Deploying Kiali to OpenShift project ${NAMESPACE}"
-# We can update the configmap/secret without updating the deployment yaml, which means our Kiali pod might not get restarted
-# We need to delete the deployment and recreate it so that we know we are getting the updated changed
+
+# We can update the configmap/secret without updating the deployment yaml, which means our Kiali pod might not get restarted.
+# We need to delete the deployment and recreate it so that we know we are getting the updated changes.
 echo "Deleting any existing Kiali deployments"
 ${OC_TOOL_PATH} delete deployment --selector=app=kiali
+
 # Only deploy the secret if we are using the login AUTH_STRATEGY
 if [ "${AUTH_STRATEGY}" == "login" ]; then
   apply_resource secret
   if [ "$?" != "0" ]; then
     echo "ERROR: Failed to deploy the Kiali secret. Aborting."
     exit 1
   fi
+else
+  echo "Using strategy [${AUTH_STRATEGY}] - a secret is not needed so one will not be created."
```

Comment:
next time you make an edit:
" a secret is not needed so one will not be created" ->  "a secret is not needed so none will be created."

Reply:
done